app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb

```diff
@@ -9,7 +9,7 @@
 
         <%= render Content::ChecklistCollageComponent.new(
           checklist: [
-            "You will earn at least £28k (or more in London) if you're just starting as a teacher.",
+            "You'll earn at least £28k (or more in London) if you're just starting as a teacher in a maintained primary, secondary or special school in England (schools funded by local authorities).",
```

Comment:
@EllieNodder I wonder if this is a bit overkill for the landing page? The advertising team don't include this caveat in their ads

We link through to the salaries and benefits page which has this detail so we are not hiding it

Reply:
worth leaving the landing page as is then?